programs/locker/src/state/claim_status.rs

```diff
@@ -0,0 +1,34 @@
+use crate::*;
+
+use safe_math::SafeMath;
+#[account]
+#[derive(Default, InitSpace, Debug)]
+pub struct ClaimStatus {
+    /// total claimed amount of recipient
+    pub total_claimed_amount: u64,
+    /// escrow address
+    pub escrow: Pubkey,
+    /// recipient address
+    pub recipient: Pubkey,
+    /// initialized
+    pub initialized: bool,
```

Comment:
why we need the field initialized?

Reply:
`initialized` used to indicate that the account's data has been initialized, and we don't need to refill again.

Reply:
we can just check claim_status. escrow == Pubkey::default to know whether the escrow is initialized or now, dont need this flag

Reply:
updated